app/views/contexts/calculated_summary_context.py

```diff
@@ -26,6 +26,7 @@
     QuestionnaireSchema,
```

Comment:
wont let me highlight above, but as you have changed the types to no longer use `Union` and `Optional` you also need to remove those from the typing imports